cppcache/include/geode/AuthenticatedView.hpp

```diff
@@ -90,7 +90,7 @@ class APACHE_GEODE_EXPORT AuthenticatedView : public RegionService {
   /**
    * @brief destructor
    */
-  virtual ~AuthenticatedView();
+  virtual ~AuthenticatedView() override;
```

Comment:
`virtual` is redundant when `override` is specified. Under some analyzers it will warn or error about it.

Reply:
Given that this is a public header, what impact is there to the ABI?

Reply:
In this case the change is not breaking ABI BC as override is just a keyword introduced in the standard in order to have the compiler checked that the virtual function you are overriding has the same signature as the one on the base class.

Reply:
whoops, missed that on my pass.  That's what reviews are for